src/item/Banner.php

```diff
@@ -66,7 +66,7 @@ public function setColor(DyeColor $color) : self{
 	}
 
 	public function getMeta() : int{
-		return DyeColorIdMap::getInstance()->toInvertedId($this->color);
+		return $this->illagerPattern ? 15 : DyeColorIdMap::getInstance()->toInvertedId($this->color);
```

Comment:
That's even worse. Now it will always show as white regardless of the server-side colour, even if the client starts to support this in the future.

Reply:
What's the expected behavior?